app/Services/WikibaseAPIClient.php

```diff
@@ -45,11 +45,20 @@ private function get(string $action, array $params): Response
         return $response;
     }
 
-    public function parseValue(string $property, $value): Response
+    /**
+     * Make a direct wbparsevalue API request.
+     * You should probably use {@link parseValues} instead.
+     *
+     * @param string $property property ID
+     * @param string[] $values up to 50
+     * @return Response
+     * @throws WikibaseValueParserException If one of the values cannot be parsed.
+     */
+    public function parseValuesForProperty(string $property, array $values): Response
     {
         try {
             return $this->get('wbparsevalue', [
-                'values' => $value,
+                'values' => implode('|', $values),
```

Comment:
In theory yes, though the existing methods don’t do it either. But I can add a commit that adds proper escaping. (I think a separate follow-up pull request might make more sense, since it’ll apply to the other methods too.)

Reply:
Done: #452

Reply:
Cool thanks! (resolving it here, further review there)